aspnetcore/security/key-vault-configuration/sample/Markup.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace KeyVaultConfigProviderSample
+{
+    internal static class Markup
+    {
+        internal const string Text = @"<!DOCTYPE html>

```

Comment:
@tdykstra @Rick-Anderson If the "poor man's templating" is a problem, see if you would allow me to go ahead and make this into a small MVC sample to preserve the output. I could work that up in a jiffy.